test/jqLiteSpec.js

```diff
@@ -612,6 +612,14 @@ describe('jqLite', function() {
       expect(elm.attr('readOnly')).toBeUndefined();
       expect(elm.attr('disabled')).toBeUndefined();
     });
+
+    it('should ignore nodes that cannot have attributes', function() {
```

Comment:
should do nothing when trying to set an attribute to a comment text node or attribute node

and please add the tests for all of them